source/operator.rst

```diff
@@ -396,6 +560,36 @@ configuration options for the HAProxy service.
 +-----------------+-------------------------------------------------------------------------------------------+
 |                                                                                                             |
 +-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _orchestrator-affinity-antiaffinitytopologykey:                                        |
+|                 |                                                                                           |
+| **Key**         | `orchestrator.affinity.antiAffinityTopologyKey                                            |
+|                 | <operator.html#orchestrator-affinity-antiAffinityTopologyKey>`_                           |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value**       | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``kubernetes.io/hostname``                                                                |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Description** | The Operator `topology key                                                                |
+|                 | <https://kubernetes.io/docs/concepts/configuration/assign-pod-node/                       |
+|                 | #affinity-and-anti-affinity>`_ node anti-affinity constraint                              |
++-----------------+-------------------------------------------------------------------------------------------+
+|                                                                                                             |
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _orchestrator-affinity-advanced:                                                       |
+|                 |                                                                                           |
+| **Key**         | `orchestrator.affinity.advanced                                                           |
+|                 | <operator.html#orchestrator-affinity-advanced>`_                                          |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value**       | subdoc                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     |                                                                                           |
```

Comment:
just reading what we have here I don't understand what does this parameter do. Can you please provide an example?